src/ExpressionEvaluator/Core/Source/ResultProvider/Formatter.cs

```diff
@@ -173,6 +175,34 @@ string IDkmClrFullNameProvider.GetClrExpressionForThis(DkmInspectionContext insp
             return _thisString;
         }
 
+        string IDkmClrFullNameProvider2.GetClrNameForLocalVariable(DkmInspectionContext inspectionContext, DkmClrModuleInstance moduleInstance, DkmClrMethodId methodId, DkmILRange iLRange, DkmClrLocalVariable localVariable)
+        {
+            return GetOriginalLocalVariableName(localVariable.Name);
+        }
+
+        string IDkmClrFullNameProvider2.GetClrNameForField(DkmInspectionContext inspectionContext, DkmClrModuleInstance moduleInstance, int fieldToken)
+        {
+            var import = (IMetadataImport)moduleInstance.GetMetaDataImport();
```

Comment:
>IMetadataImport [](http://example.com/codeflow?start=26&length=15)

Where is `IMetadataImport` defined?

Reply:
It's in Microsoft.MetadataReader namespace, in Microsoft.VisualStudio.Debugger.Metadata.dll AKA VIL